rerun_py/rerun_sdk/rerun/_properties.py

```diff
@@ -0,0 +1,47 @@
+from __future__ import annotations
+
+import rerun_bindings as bindings
+
+from .recording_stream import RecordingStream
+
+
+def set_properties(properties: bindings.RecordingProperties, recording: RecordingStream | None = None) -> None:
+    """
+    Set the properties of the recording.
+
+    These are builtin properties of the Rerun Viewer.
```

Comment:
sounds like they are properties of the viewer, the application, itself. How about

```suggestion
    These are builtin recording properties known to the Rerun Viewer.
```